src/core/crypto_tls.c

```diff
@@ -387,7 +386,21 @@ QuicCryptoTlsReadExtensions(
                 return Status;
             }
 
-        } else if (ExtType == TlsExt_QuicTransportParameters) {
+        } else if (
+            Connection->Stats.QuicVersion != QUIC_VERSION_DRAFT_29 &&
```

Comment:
we're sure we'll never get into the code path except when the version is 29 or 1?